website/src/templates/markdown.jsx

```diff
@@ -18,6 +21,7 @@ type Props = {
 
 export default ({ data, location }: Props) => (
   <Layout location={location}>
+    <a href={data.markdownRemark.fields.gitUrl}>Edit these docs</a>
```

Comment:
Can you please add the edit button to the bottom of the page?

What do you think?